src/components/bc-pay-button.ts

```diff
@@ -60,7 +60,7 @@ export class PayButton extends withTwind()(BitcoinConnectElement) {
     return html` <div class="inline-flex" @click=${this._onClick}>
       <bci-button variant="primary">
         ${isLoading
-          ? html`<span class="ml-1 mr-1">${loadingIcon}</span>`
+          ? html`${waitingIcon(`w-10 h-10`)}`
```

Comment:
the margins were removed, does it still match without any jumping when the state changes?

Reply:
I don't see a difference when the state changes but I'm gonna add them again anyway